client/src/components/Footer.jsx

```diff
@@ -2,29 +2,27 @@ import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 
 var FontAwesome = require('react-fontawesome');
+
 class Footer extends Component {
```

Comment:
You should convert this component to a [stateless component](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc) since it (has no state) and to avoid unnecessary rerenders!